setup/install/providers/aws/aws-ec2.md

```diff
@@ -119,21 +134,24 @@ hal config provider aws account add $AWS_ACCOUNT_NAME \
     --assume-role role/spinnakerManaged
 ```
 
-Now enable AWS
+### Enable the Spinnaker AWS provider
 
 ```bash
 hal config provider aws enable
 ```
 
 ## Advanced account settings
+Once you configure the Spinnaker AWS provider you will be able to manage and deploy EC2 resources with Spinnaker.
```

Comment:
```suggestion
After you configure the Spinnaker AWS provider you can manage and deploy EC2 resources with Spinnaker.
```

Reply:
Thanks. More proper English there. Thank you.